src/components/Collapsible/Collapsible.test.js

```diff
@@ -4,62 +4,65 @@ import {expect} from 'chai';
 import sinon from 'sinon';
 import Collapsible from './Collapsible';
 
-const waitFor = delay => new Promise(resolve => setTimeout(resolve, delay));
-
 describe('<Collapsible/>', () => {
```

Comment:
Looks like somehow we lost 2% coverage (based on codecov), can we try to cover whatever's missing?

Reply:
I have no idea how, I even added a test :) so weird. Now there's less code + extra test, but less coverage? so weird

Reply:
added 1 extra test